isamples_metadata/OpenContextTransformer.py

```diff
@@ -0,0 +1,217 @@
+import typing
+
+from isamples_metadata.Transformer import (
+    Transformer,
+    AbstractCategoryMetaMapper,
+    StringEqualityCategoryMapper,
+    AbstractCategoryMapper,
+)
+
+
+class MaterialCategoryMetaMapper(AbstractCategoryMetaMapper):
+    _anthropogenicMaterialMapper = StringEqualityCategoryMapper(
+        [
+            "Architectural Element",
+            "Bulk Ceramic",
+            "Glass",
+            "Object",
+            "Pottery",
+            "Sample",
+            "Sample, Collection, or Aggregation",
+            "Sculpture",
+            "Stela",
+        ],
+        "Anthropogenic material",
+    )
+
+    _anthropogenicMetalMapper = StringEqualityCategoryMapper(
+        [
+            "Coin",
+        ],
+        "Anthropogenic metal",
+    )
+
+    _biogenicMapper = StringEqualityCategoryMapper(
+        ["Animal Bone", "Human Bone", "Non Diagnostic Bone", "Shell"],
+        "Biogenic non organic material",
+    )
+
+    _organicMapper = StringEqualityCategoryMapper(
+        [
+            "Biological subject, Ecofact",
+            "Plant remains",
+        ],
+        "Organic material",
+    )
+
+    _rockMapper = StringEqualityCategoryMapper(["Groundstone"], "Rock")
+
+    @classmethod
+    def categories_mappers(cls) -> typing.List[AbstractCategoryMapper]:
+        return [
+            cls._anthropogenicMaterialMapper,
+            cls._anthropogenicMetalMapper,
+            cls._biogenicMapper,
+            cls._organicMapper,
+            cls._rockMapper,
+        ]
+
+
+class SpecimenCategoryMetaMapper(AbstractCategoryMetaMapper):
+    _organismPartMapper = StringEqualityCategoryMapper(
+        [
+            "Animal Bone",
+            "Human Bone",
+            "Non Diagnostic Bone",
+        ],
+        "Organism part",
+    )
+    _anthropogenicAggregationMapper = StringEqualityCategoryMapper(
+        ["Architectural Element", "Basket", "Bulk Ceramic", "Lot"],
```

Comment:
I think "Architectural Element" and "Bulk Ceramic" are OK here. But "Basket" and "Lot" are types of sub-site contexts / locations where stuff was found.

Reply:
@smrgeoinfo Any suggestions on where "Basket" and "Lot" might better map?

Reply:
Opened https://github.com/isamplesorg/metadata/issues/53 to sort this one out.  We can fix the code up if necessary after we get confirmation.  Thanks @ekansa!